services/app-api/src/testHelpers/contractAndRates/contractHelpers.ts

```diff
@@ -9,20 +9,30 @@ import type {
 } from '../../postgres/contractAndRates/prismaSubmittedContractHelpers'
 import type { StateCodeType } from 'app-web/src/common-code/healthPlanFormDataType'
 import type { ContractFormDataType } from '../../domain-models/contractAndRates'
+import type { ProgramType } from '../../domain-models'
+import statePrograms from 'app-web/src/common-code/data/statePrograms.json'
+
+function getProgramsFromState(stateCode: StateCodeType): ProgramType[] {
```

Comment:
Good idea for reusable function - I see this code all over. 
 
 Nit: file organization -  this looks like a stateHelper not a contractHelper.

Reply:
This is your reusable function! I just took it and put it in here. I will extract them all out and put them in a `stateHelpers.ts` file.